### PR TITLE
docs: add diagrams for the import and scanning component

### PR DESCRIPTION
**Pull request by namhoang1604**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

This PR updates the diagram for the import and scanning component.

## Related Tickets & Documents

https://github.com/gs-gs/fa-ag-trace/issues/667

## Mobile & Desktop Screenshots/Recordings

image
image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [x] 📕 documentation
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: documentation/docs/mock-apps/components/qr-code-scanner-dialog-button.md [hunk 1]
```diff
@@ -23,6 +23,31 @@ The QRCodeScannerDialogButton component is responsible for rendering a button th
...
+## Diagram
+
+```mermaid
+sequenceDiagram
+    participant U as User
+    participant MA as Mock App
+    participant V as VCKit
+
+    U->>MA: Scan QRCode
+    MA->>MA: Fetch data from QRCode URL
```
Comment (ashleythedeveloper) [inline documentation/docs/mock-apps/components/qr-code-scanner-dialog-button.md:35]: @namhoang1604, what are the permitted data structures of the QRpayload? Can we also use a Data Matrix? What are the permitted data structures of the file being fetched? What are the allowed file extensions? What are the permitted credential types? What happens if the contents of the credential being fetched are encrypted?

**Review by ashleythedeveloper — CHANGES_REQUESTED** [review]

File: documentation/docs/mock-apps/components/import-button.md [hunk 2]
```diff
@@ -23,6 +24,30 @@ The ImportButton component is responsible for rendering a button that allows the
...
 
+## Diagram
+
+```mermaid
+sequenceDiagram
+    participant U as User
+    participant MA as Mock App
+    participant V as VCKit
+
+    U->>MA: Import file
```
[the remark this answers is not in the document]
Comment (namhoang1604) [reply documentation/docs/mock-apps/components/import-button.md:35]: Hi @ashleythedeveloper, the data structure is defined [here](https://github.com/uncefact/tests-untp/blob/1fc228a6b367a70b4e1c5a07b2dc4707f30c78ae/documentation/docs/mock-apps/components/import-button.md#props), they can be VC or JSON. There is a component [render-check-list](https://github.com/uncefact/tests-untp/blob/1fc228a6b367a70b4e1c5a07b2dc4707f30c78ae/documentation/docs/mock-apps/components/render-check-list.md#response-data) added examples for cases can happen.

File: documentation/docs/mock-apps/components/qr-code-scanner-dialog-button.md (hunk 1 above)
Comment (namhoang1604) [reply documentation/docs/mock-apps/components/qr-code-scanner-dialog-button.md:35]: The data structure is defined [here](https://github.com/uncefact/tests-untp/blob/1fc228a6b367a70b4e1c5a07b2dc4707f30c78ae/documentation/docs/mock-apps/components/qr-code-scanner-dialog-button.md#props), it allows VC or JSON. The current implementation is not handling the encrypted data.

File: documentation/docs/mock-apps/components/import-button.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (namhoang1604) [reply documentation/docs/mock-apps/components/import-button.md:35]: Thanks @ashleythedeveloper, I updated it for more details.

File: documentation/docs/mock-apps/components/qr-code-scanner-dialog-button.md (hunk 1 above)
Comment (namhoang1604) [reply documentation/docs/mock-apps/components/qr-code-scanner-dialog-button.md:35]: Thanks @ashleythedeveloper, I updated it for more details.

**Review by ashleythedeveloper — APPROVED** [review]

**Review by ldhyen99 — APPROVED** [review]

**Review by huynguyen-hl — APPROVED** [review]

**Pull request merged**